zip-0316.rst

```diff
@@ -664,6 +769,12 @@ Change level is non-hardened.
 Deriving a UIVK from a UFVK
 ---------------------------
 
+As of `Revision 1`_, a Consumer of a UIVK MUST understand the
+meaning of all "MUST-understand" Metadata Item Typecodes present in the
+UFVK in order to perform UIVK derivation. For Metadata Items recognised
```

Comment:
Express this as a consequence of the general MUST-understand requirement rather than restating it as if it were a separate conformance requirement.